test/helpers/random_number_string.go

```diff
@@ -0,0 +1,14 @@
+package helpers
+
+import (
+	"math/rand"
+	"strconv"
+)
+
+// RandomNumberString provides a string containing the given amount of random numbers.
```

Comment:
Thoughts on using go uuid or making this deterministic (ie use a global counter that goes up)? Had some tests fail due to random numbers clashing

Reply:
Go is so cool. I'm using `atomic.AddUint32` to implement this now in a new `UniqueString` helper. Great idea!